rspace/src/main/scala/coop/rchain/rspace/internal.scala

```diff
@@ -13,6 +13,9 @@ import scala.collection.SortedSet
 @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
 object internal {
 
+  /** helper class to package data and serialization (encoded by scodec) */
+  final case class Encoded[D](item: D, byteVector: ByteVector)
```

Comment:
Having ByteVector next to Scala object helps greatly when doing diffs on pair of vectors.

Reply:
Helps in which way? Performance?

Reply:
@zsluedem yes, its faster